end_users/README.md

```diff
@@ -1,3 +1,24 @@
 # End Users
 
-Documentation coming soon...
+## Table of Contents
+
+* [Merge Options](#merge-options)
+  * [Setting the Correct Merge Option](#setting-the-correct-merge-option)
+
+## Merge Options
+
+Repositories that use BonnyCI should not use any merge strategy which changes the SHA of the commit being merged in. The reasoning for this is that [Zuul](https://github.com/openstack-infra/zuul), the scheduler used in BonnyCI, will be thrown off by the change in SHA when it merges commits that successfully pass the gate queue.
+
+GitHub presents repository owners with 3 choices for what happens when a commit is merged from a pull request:
+
+* Create a merge commit
+* Squash and merge
+* Rebase and merge
+
+For more on these options and what they mean, read the [documentation on merge methods](https://help.github.com/articles/about-merge-methods-on-github/) provided by GitHub.
+
+### Setting the Correct Merge Option
+
+In order for BonnyCI to function correctly, please ensure that your repository only allows merge commits for its merge method. From your repository's home page, navigate to the `Settings` tab. Scroll down to the section labeled `Merge button`, and ensure that it matches the following:
```

Comment:
In order for BonnyCI to function correctly, please ensure that your repository only allows merge commits.